curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -410,16 +418,29 @@ MetaStatusCode InodeStorage::ModifyInodeS3ChunkInfoList(
     const S3ChunkInfoList* list2del) {
     WriteLockGuard lg(rwLock_);
     auto txn = kvStorage_->BeginTransaction();
+    uint32_t step = 0;
```

Comment:
you can use a meaningful const variables to replace `step` variable.

Reply:
fixed